torch_xla/csrc/tensor_methods.cpp

```diff
@@ -953,6 +953,16 @@ XLATensor XLATensor::eye(xla::int64 lines, xla::int64 cols,
       device, element_type);
 }
 
+void XLATensor::eye_out(XLATensor& out, xla::int64 lines, xla::int64 cols) {
+  if (cols < 0) {
```

Comment:
In these cases, I tend to like more: `cols >= 0 ? cols : lines` directly as function argument below.

Reply:
Haha in `eye`'s case, `eye_out` is the leaf node (where we dispatch to different devices). 
Yea PT is not consistent in this case, our integration will help clean up some in the future ;)

Reply:
oops wrong place for the reply :P 
Yea I'll fix the new comments :D